docs/investigations/graphql-partial-updates/graphql-partial-updates.md

```diff
@@ -0,0 +1,156 @@
+# GraphQL Partial Updates
```

Comment:
to check: If some string field is required, can I provide "" as its value?

Reply:
Yes, `""` is a valid string value for a required field.